scripts/pysa_fuzzer/run.py

```diff
@@ -0,0 +1,148 @@
+import subprocess
+import os
+import json
+import argparse
+from code_generator import CodeGenerator
+from pathlib import Path
+import shutil
+import logging
+
+logging.basicConfig(level=logging.INFO)
+
+def run_command(command, output_file=None):
+    try:
+        result = subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True, check=True)
+        if output_file:
+            with open(output_file, 'w') as file:
+                file.write(result.stdout)
+    except subprocess.CalledProcessError as e:
+        logging.error(f"Command '{command}' failed with error: {e.stderr}")
+
+def generate_python_files(num_files, num_statements):
+    generator = CodeGenerator()
+    output_dir = Path('generated_files')
+
+    # Create the directory if it doesn't exist

```

Comment:
nitpick: This comment doesn't add much value, since `mkdir` and `exist_ok` makes it clear what this is doing. We could remove this.